### PR TITLE
[B5] settings: change my password page

### PR DESCRIPTION
**Pull request by orangejenny**
## Product Description
Working through the `settings` app. This is an easy one.

before
Screenshot 2024-04-22 at 8 18 00 AM

after
Screenshot 2024-04-22 at 8 23 18 AM

### Safety story
Minor changes, review should be sufficient.

### Automated test coverage

no

### QA Plan

not requesting QA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by orangejenny** [comment]
I'm not planning to fix the pre-existing lint warnings.

File: corehq/apps/settings/forms.py [hunk 1]
```diff
@@ -54,13 +54,11 @@ def __init__(self, user, *args, **kwargs):
                     data_bind="value: password, valueUpdate: 'input'",
                 ),
                 'new_password2',
-                hqcrispy.FormActions(
```
Comment (millerdev) [inline corehq/apps/settings/forms.py:59]: Out of curiosity, why was this removed? Is it no longer needed in B5?

**Review by millerdev — APPROVED** [review]

File: corehq/apps/settings/forms.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/settings/forms.py:59]: It's still in B5, and it's still in our docs for two-column forms ([here](https://www.commcarehq.org/styleguide/b5/organisms/forms/#crispy-forms-simple)). I've been pulling it out of vertical forms as a simplification. With the submit button now all the way to the left, it's naturally what the user sees as they finish the form.

I thought removing form actions was our standard for vertical forms, but now I'm realizing that we don't have a documented standard, I've been working off of the [maintenance alerts page](https://staging.commcarehq.org/alerts/) (one of the first pages that was migrated to B5, which uses vertical forms).

FYI @biyeun in case you have thoughts on what our standards for vertical forms should be.
